ext/console/02_console.js

```diff
@@ -1284,15 +1283,9 @@
     } else if (ObjectPrototypeIsPrototypeOf(DatePrototype, value)) {
       return inspectDate(value, inspectOptions);
     } else if (ObjectPrototypeIsPrototypeOf(SetPrototype, value)) {
-      return inspectSet(
-        proxyDetails ? proxyDetails[0] : value,
```

Comment:
Maybe i should write code like this in Date branch?

Reply:
I'd accept that change too.

Reply:
I have modified it according to this.